CHANGELOG.md

```diff
@@ -1,6 +1,52 @@
+# 1.4.10 (2023-05-01)
+
+Beginning with this release we are publishing installable packages for Red Hat Enterprise Linux 9 (amd64) on Microsoft's [Linux package repository](https://packages.microsoft.com/docs/readme.txt).
+
+**Note:** On RHEL 9 the IoT Edge security subsystem has been tested with openssl 3.0. It may not function properly if older versions of openssl are also present on the device. If you previously installed openssl 1.1 in combination with an earlier version of IoT Edge then we would recommend removing both and starting fresh to avoid potential incompatibilities.
+
+## Base image updates
+
+The following Docker images were updated because their base images changed:
+* azureiotedge-agent
+* azureiotedge-hub
+* azureiotedge-simulated-temperature-sensor
+* azureiotedge-diagnostics
+
+## Edge Agent
+### Bug fixes
+* Update versions of .NET dependencies to patch security vulnerabilities ( [82ca5e8](https://github.com/Azure/iotedge/commit/82ca5e8eb082a3d5ebf8276fc0109f923c022ad2) )
+* Update version of Azure IoT SDK to fix a memory leak ( [d98f43c](https://github.com/Azure/iotedge/commit/d98f43c90d456f51903ee2fa3f7266083086d739) )
+* Optionally detect and remove orphaned module identities on startup ( [3bac802](https://github.com/Azure/iotedge/commit/3bac80274305c7f2d4af92c161bead2c486d6820) )
```

Comment:
What is the verbage we normally use for when EdgeAgent brings modules up? startup or deployment?

Reply:
A deployment happens whenever edge agent receives a twin update that changes the config. I don't think there's a precise tech term for when edge agent is first starting up (including reconciling the config).

Reply:
So, the check happens on receiving twin, so maybe deployment will be more accurate term to use here then.

Reply:
Done